dss/util/auth/__init__.py

```diff
@@ -2,18 +2,17 @@
 
 from dss import Config
 from dss.error import DSSException
-from . import fusillade
+from . import fusillade, authregistry
 
 logger = logging.getLogger(__name__)
 
 
 class AuthHandler:
     def __new__(cls, *args, **kwargs):
         auth_backend = Config.get_auth_backend()
```

Comment:
```suggestion
        auth_backend = Config.get_auth_backend().lower()
```

This avoids confusion about whether auth handler registry is case sensitive, required syntax, etc. 

(Alternatively, can do the `.lower()` call in the Config class, but this seems like the most appropriate place to do it since we're just trying to make the registry case-insensitive)

Reply:
i opted to add this version, rather than make the changes to the Config class